### ISSUE TITLE
unstable direct pre-training on Retinanet with different seeds

### ISSUE THREAD

**Issue by belisarius0** [wxzs5/direct-pretraining#7]
In retinenet's direct pre-train, I observed that loss becomes nan or inf during warmup when some seeds are specified for train.py.
Wrt default seed, No problem.
I hope to re-explore the hyper parameters to make the learning more stable.
